3_visualize/src/plot_radial.R

```diff
@@ -25,40 +32,46 @@ plot_radial_chart <- function(major_drought_periods, drought_events,
     theme(legend.position = "none",
           axis.text = element_blank(),
           axis.title = element_blank(),
+          panel.grid.minor = element_blank(),
+          # drop extra major grid line that coord_polar adds
+          panel.grid.major.y = element_line(color = c(rep('grey85', 13), 'NA')),
           panel.grid.major.x = element_blank(),
-          panel.grid.minor.x = element_blank())+
+          panel.grid.minor.x = element_blank(),
+          plot.margin = margin(t = 0, r = 0, b = 0, l = 0, unit = "pt"))+
     # Manually add year labels
     annotate(geom = "text",
-             label = seq(1920, 2020, by = 10), 
-             x = rep(0, 11),
-             y = as.Date(sprintf("%s-01-01", seq(1920, 2020, by = 10))),
-             size = 2)
+             label = seq(1920, 2020, by = 20), 
```

Comment:
now fixed - axis labels are added to svg instead of background plot
![image](https://github.com/DOI-USGS/drought-timeline/assets/54007288/d155411b-e55f-4561-a306-1f5bf43e03d1)